src/ui/routes/packages/[name]/review/+page.svelte

```diff
@@ -0,0 +1,69 @@
+<!-- 
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+ -->
+<script lang="ts">
+	import {
+		PackageDetailsCard as PackageDetails,
+		PackageComponentAccordion as PackageComponent,
+		PackageSectionHeader as SectionHeader,
+		Divider,
+	} from '$lib/components';
+
+	import { pkgComponentDeployStore, pkgStore } from '$lib/store';
+	import { Button, AccordionGroup, currentTheme } from '@ui';
+	import { page } from '$app/stores';
+	import BuildProvidence from '$lib/components/build-providence.svelte';
+	import DeploymentActions from '$lib/components/deployment-actions.svelte';
+</script>
+
+<svelte:head>
+	<title>Review</title>
+</svelte:head>
+
+<SectionHeader>
+	Package Details
+	<span slot="tooltip">At-a-glance simple metadata about the package</span>
+</SectionHeader>
+<PackageDetails pkg={$pkgStore.zarfPackage} />
+
+<SectionHeader icon="secured_layer">
+	Supply Chain
+	<span slot="tooltip"
+		>Supply chain is used to help determine if a package can be trusted. It includes declarative
+		data regarding how this package was built. Build providence includes metadata about the build
+		and where the package was created. SBOM includes information on all of the code, images, and
+		resources contained in this package.
+	</span>
+</SectionHeader>
+<BuildProvidence build={$pkgStore.zarfPackage.build} />
+
+<SectionHeader
+	>Components
```

Comment:
```suggestion
<SectionHeader>
  Components
```
I've noticed this across a few files, is this a stylistic thing? Normally a tag is kept together rather than having its end split onto the next line.

Reply:
This seems to be svelte issue that I have not found a solid solution for.

Reply:
I fixed it, Had to set the htmlWhitespaceSensitivity to ignore in prettierrc